### PR TITLE
Renaming new SessionRetryOptionsBuilder settings

### PR DESCRIPTION
**Pull request by FabianMeiswinkel**

### CODE REVIEW

File: sdk/cosmos/azure-cosmos/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 ### 4.52.0-beta.1 (Unreleased)
 
 #### Features Added
+* Added an option to configure the minimum retry duration for 404/1002 session not available. - See [PR 37143](https://github.com/Azure/azure-sdk-for-java/pull/37143) and [PR 37240](https://github.com/Azure/azure-sdk-for-java/pull/37240)
```
Comment (xinlian12) [inline sdk/cosmos/azure-cosmos/CHANGELOG.md:6]: since we have changed the API name, maybe here also match the new name - `minTimeoutPerRegion`

**Review by xinlian12 — APPROVED** [review]
LGTM, thanks

File: sdk/cosmos/azure-cosmos/CHANGELOG.md (hunk 1 above)
Comment (FabianMeiswinkel) [reply sdk/cosmos/azure-cosmos/CHANGELOG.md:6]: Will do in next PR

**Pull request merged**